src/mlpack/methods/nca/nca_softmax_error_function_impl.hpp

```diff
@@ -9,8 +9,8 @@
  * 3-clause BSD license along with mlpack.  If not, see
  * http://www.opensource.org/licenses/BSD-3-Clause for more information.
  */
-#ifndef MLPACK_METHODS_NCA_NCA_SOFTMAX_ERROR_FUNCTCLIN_IMPL_H
-#define MLPACK_METHODS_NCA_NCA_SOFTMAX_ERROR_FUNCTCLIN_IMPL_H
+#ifndef MLPACK_METHODS_NCA_NCA_SOFTMAX_ERROR_FUNCTCMDN_IMPL_H
+#define MLPACK_METHODS_NCA_NCA_SOFTMAX_ERROR_FUNCTCMDN_IMPL_H
```

Comment:
Looks like a bad regex here---would you mind fixing it?

Reply:
Of course, I am just wondering why it is called `FUNCTCLIN` at the first place?

Reply:
Heh, good point; that's probably because a regex was used to change `IO` to `CLI` many years ago and this was missed...